bin/app

```diff
@@ -356,9 +359,9 @@ class App < Sinatra::Base
   #
   get "/json/github/donut/allpulls" do
     pulltop = []
-    allpulls = Pullrequest.aggregate(:repository_rname, :all.count)
+    allpulls = Pullrequest.aggregate(:repository_organization_oname, :repository_rname, :all.count)
     allpulls.each do |donut_bite|
-      pulltop << {label: donut_bite[0], value: donut_bite[1]}
+      pulltop << {label: "#{donut_bite[0]}/#{donut_bite[1]}", value: donut_bite[2]}
```

Comment:
the string could get too long in the donut hole, and subsequently the font size gets too small. but that is a problem of the `morris.js` library which could lead to some ugly svg hacking.

would you be fine with not adding the org_name in front of every repo here in this view?

so just
``` ruby
pulltop << {label: "#{donut_bite[1]}", value: donut_bite[2]}
```